Source/AppAuthTV/OIDTVAuthorizationResponse.h

```diff
@@ -85,7 +85,9 @@ NS_ASSUME_NONNULL_BEGIN
     @see https://tools.ietf.org/html/rfc8628#section-3.4
  */
 - (nullable OIDTVTokenRequest *)tokenPollRequestWithAdditionalParameters:
```

Comment:
Similar question to the above. Should we add methods that take only `additionalHeaders` and only `additionalParameters`? Those could call thru to the method that takes both and pass `nil` to the parameter when they are missing the argument.

Reply:
Followed up on this the same way as for https://github.com/openid/AppAuth-iOS/pull/770#pullrequestreview-1458462192